widget/embedded/src/components/TokenList/TokenList.tsx

```diff
@@ -133,24 +133,24 @@ export function TokenList(props: PropTypes) {
           const address = token.address || '';
           const blockchain = blockchains.find(
             (blockchain) => blockchain.name === token.blockchain
+          ) as BlockchainMeta;
+          const colors = createTintsAndShades(
+            blockchain.color,
+            blockchain.shortName,
```

Comment:
you don't need to pass `shortName` here. you can use a fixed name like `main` or `primary`. it helps you to change `$$color: colors[`${blockchain.shortName}250`],` to `$$color: colors.main250`